packages/person/src/person-card/element.css.ts

```diff
@@ -0,0 +1,91 @@
+import { css, unsafeCSS } from 'lit';
+import { styles as theme } from '@equinor/fusion-web-theme';
+
+export const style = css`
+  :host {
+    font-family: ${unsafeCSS(theme.typography.paragraph.body_long.getVariable('fontFamily'))};
+    font-size: calc(
+      ${unsafeCSS(theme.typography.paragraph.body_long.getVariable('fontSize'))} * var(--content-resize, 1)
+    );
+  }
+  :host header {
```

Comment:
element selector within shadow-dom do not require prefix of `:host`, they are implicit scoped within the host